docs/getting-started.md

```diff
@@ -210,7 +210,7 @@ kubectl describe vpc vpc-0f54c9f1b395038ab -n sample-ns
 
 ```text
 # Output
-Name:         vpc-0f54c9f1b395038ab
+Name:         vpc-0d6bb6a4a880bd9ad
```

Comment:
Also, can we update line 208 to have vpc-0d6bb6a4a880bd9ad instead of vpc-0f54c9f1b395038ab?

Reply:
Done